Text/CustomEntitySearch/CustomEntitySearch.cs

```diff
@@ -50,11 +51,11 @@ public static class CustomEntitySearch
 
             WebApiSkillResponse response = WebApiSkillHelpers.ProcessRequestRecords(skillName, requestRecords,
                 (inRecord, outRecord) => {
-                    string text = Regex.Escape(inRecord.Data["text"] as string);
+                    string text = inRecord.Data["text"] as string;
                     List<string> words = ((JArray)inRecord.Data["words"]).ToObject<List<string>>();
                     if (words == null)
                     {
-                        words = new WordLinker(executionContext.FunctionAppDirectory).Words;
+                        words = new List<string>(new WordLinker(executionContext.FunctionAppDirectory).Words);
```

Comment:
No need to allocate a new list: this is already what you're getting out of `Words`.